README.md

```diff
@@ -57,6 +57,23 @@ cur = conn.cursor()
 cur.execute('SELECT * FROM system.runtime.nodes')
 rows = cur.fetchall()
 ```
+# JWT Authentication
+The `JWTAuthentication` can be used to connect to a JWT token authentication enabled Trino cluster:
+```python
+import trino
+conn=trino.dbapi.connect(
+    host='coordinator-url',
+    port=8443,
+    user='the-user',
+    catalog='the-catalog',
+    schema='the-schema',
+    http_schema='https',
+    auth=trino.auth.JWTAuthentication("jwt-token"),
```

Comment:
Could we sub the "jwt-token" example string for an actual JWT token? e.g.  `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiSm9obiBEb2UifQ.DjwRE2jZhren2Wt37t5hlVru6Myq4AhpGLiiefF69u8`? As is, it looks like a string based config at first glance. If not, maybe angle brackets?

Reply:
How about this @joshthoward 

```python
import trino
JWT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiSm9obiBEb2UifQ.DjwRE2jZhren2Wt37t5hlVru6Myq4AhpGLiiefF69u8 
conn=trino.dbapi.connect(
    host='coordinator-url',
    port=8443,
    user='the-user',
    catalog='the-catalog',
    schema='the-schema',
    http_schema='https',
    auth=trino.auth.JWTAuthentication(JWT_TOKEN),
)
cur = conn.cursor()
cur.execute('SELECT * FROM system.runtime.nodes')
rows = cur.fetchall()
```

Reply:
That works (just quote it haha)